src/files/ls.js

```diff
@@ -19,7 +19,7 @@ module.exports = configure(({ ky }) => {
     searchParams.set('arg', CID.isCID(path) ? `/ipfs/${path}` : path)
     searchParams.set('stream', true)
     if (options.cidBase) searchParams.set('cid-base', options.cidBase)
-    if (options.long != null) searchParams.set('long', options.long)
+    searchParams.set('long', options.long == null ? true : options.long)
```

Comment:
In JS this information is actively available and we jump through hoops to remove it from the response (even though it's not really removed, it's just set to default values - empty string, zero etc.). I've removed it from the core interface so that we always get the information as I believe this option is actually useful for CLI formatting.